dev-docs/bidders/smartadline.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: bidder
+title: smartadline
+description: Prebid Smartadline Bidder Adaptor
+pbjs: false 
```

Comment:
Surely you support either `pbjs` or `pbjs`?

Reply:
@fowler446 , thank you for your feedback!
I've fixed it here: https://github.com/prebid/prebid.github.io/pull/4203/commits/8baf8eeed2d2129c3b6f5d24e0bc5f122f6164c3

Reply:
@fowler446 , please can you give me any feedback? Should I need to fix something else here?